### DISCUSSION TITLE
Aliasing gap -> g as padding -> p

### DISCUSSION THREAD

**Discussion by 3xau1o** [tailwindlabs/tailwindcss#12857] [Help] [answered]
Would like to be able to set gaps using `g-1` `g-2` similar to padding and marging classes

**Comment by makibat** [comment, marked as answer]
I guess the easiest way to achieve that would be by writing a [plugin](https://tailwindcss.com/docs/plugins) which utilizes the default values: 
``` js
plugins: [
    plugin(function ({ matchUtilities, theme }) {
      matchUtilities(
        {
          g: (value) => {
            return {
              gap: value,
            }
          },
        },
        { values: theme('gap') }
      )
    }),
  ]
```

And then you could set gaps as `g-10` for example or even with arbitrary values like `g-[2rem]`
Here's a [Tailwind Play](https://play.tailwindcss.com/N8xLA4CVzf)
